### PR TITLE
Use exec instead of a subprocess in sandbox

### PR DESCRIPTION
**Pull request by peterebden**
Came across this while I was looking into its performance; this doesn't make a lot of difference to that but I think it's better not to create an additional process. Also has some mild advantages like signal forwarding and propagating the correct exit code.

### CODE REVIEW

**Review by tiagovtristao — APPROVED** [review]
The change makes sense to me, thanks.

File: src/sandbox/sandbox_linux.go [hunk 1]
```diff
@@ -65,22 +66,19 @@ func Sandbox(args []string) error {
 			return fmt.Errorf("Failed to bring loopback interface up: %s", err)
 		}
 	}
-
-	return cmd.Run()
+	err = syscall.Exec(cmd, args, env)
+	return fmt.Errorf("Failed to exec %s: %s", cmd, err)
```
Comment (PeterBocan) [inline src/sandbox/sandbox_linux.go:70]: do you want to always return an error?

Comment (tiagovtristao) [reply src/sandbox/sandbox_linux.go:70]: An error will only be returned if it failed to execute `cmd`. I think `syscall.Exec` uses `execve` under the hood which replaces the current process, so this function would never return on success.

Comment (peterebden) [reply src/sandbox/sandbox_linux.go:70]: Indeed :D

**Review by samwestmoreland — APPROVED** [review]
Looks good to me!

**Pull request merged**